service/src/main/resources/application.properties

```diff
@@ -1,8 +1,23 @@
+#App Config
 server.port = 9090
 spring.application.name = login-gateway
+
+#Rest General Config
 logingw.rest.config.some-key = BETA
 logingw.rest.config.alg-name = RS256
 logingw.rest.config.exp-time = 2
+
+#Rest Auth Config
 logingw.rest.auth.ad.ldap.domain = some.domain.com
 logingw.rest.auth.ad.ldap.url = ldaps://some.domain.com:636/
 logingw.rest.auth.ad.ldap.search-filter = (samaccountname={1})
+
+#Health Check Config
+management.endpoints.web.exposure.include=health
+#TODO: Enable Ldap check when fully Implemented - issue #20
+management.health.ldap.enabled=false
+management.endpoint.health.show-details=never
+
+#JWX Config
+spring.jmx.enabled=true
+management.endpoints.jmx.exposure.include=health
```

Comment:
Also, consider adding
```properties
springdoc.show-actuator=true
```
in order for the actuator endpoint to appear as part of the Swagger UI, please.